keyboards/keychron/q3/readme.md

```diff
@@ -0,0 +1,63 @@
+# Keychron Q3
```

Comment:
This file is missing description and flashing instructions for `rev_0120_lhg`.

Reply:
Thanks a lot, because it is for the ansi version as the rev_0120 so i ignored it, i add them now.